### ISSUE TITLE
Override the error popup windows in Angular

### ISSUE THREAD

**Issue by tang-butian** [abpframework/abp#7152]
![image](https://user-images.githubusercontent.com/27254047/104261235-0f2d1980-54c0-11eb-8120-8af18becc1ec.png)
How can I override the RestService error handling;  Is not disabled

**Comment by mehmet-erim** [comment]
Hello,

You can override the [`ErrorHandler`](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts) easily. 

1. Create a new service. You can create as the following command:

```bash
yarn ng generate service custom-error-handler
```

2. Add a provide to the `AppModule` as shown below:

```ts
// app.module.ts

//...
@NgModule({
  //...
  providers: [..., { provide: ErrorHandler, useClass: CustomErrorHandlerService }],
```

ErrorHandler can be overridden in just 2 steps. You can copy the [error.handler.ts`](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts) content to your custom error handler. Then you can change the methods according to how you want to show errors.

Note: Change relative path imports like this

```
import {
  HttpErrorWrapperComponent,
  ErrorScreenErrorCodes,
  HttpErrorConfig,
  Confirmation,
  ConfirmationService,
} from '@abp/ng.theme.shared';
```

Note 2: You will see an importing error about the `HttpErrorWrapperComponent`. This will be fixed in v4.1.1. If you would not like to wait for v4.1.1, you can create a custom component for the `HttpErrorWrapperComponent`.

**Comment by tang-butian** [comment]
Thank you very much

**Comment by pouyababaie** [comment]
> Hello,
> 
> You can override the [`ErrorHandler`](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts) easily.
> 
> 1. Create a new service. You can create as the following command:
> 
> ```shell
> yarn ng generate service custom-error-handler
> ```
> 
> 2. Add a provide to the `AppModule` as shown below:
> 
> ```ts
> // app.module.ts
> 
> //...
> @NgModule({
>   //...
>   providers: [..., { provide: ErrorHandler, useClass: CustomErrorHandlerService }],
> ```
> 
> ErrorHandler can be overridden in just 2 steps. You can copy the [error.handler.ts`](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts) content to your custom error handler. Then you can change the methods according to how you want to show errors.
> 
> Note: Change relative path imports like this
> 
> ```
> import {
>   HttpErrorWrapperComponent,
>   ErrorScreenErrorCodes,
>   HttpErrorConfig,
>   Confirmation,
>   ConfirmationService,
> } from '@abp/ng.theme.shared';
> ```
> 
> Note 2: You will see an importing error about the `HttpErrorWrapperComponent`. This will be fixed in v4.1.1. If you would not like to wait for v4.1.1, you can create a custom component for the `HttpErrorWrapperComponent`.

is there a way to extend or implement that class to an  Injectable service?
